ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -535,4 +537,30 @@ - (RTCMediaStreamTrack *)trackForId:(nonnull NSString *)trackId pcId:(nonnull NS
     return peerConnection.remoteTracks[trackId];
 }
 
+- (void)ensureAudioSessionWithRecording {
+  RTCAudioSession* session = [RTCAudioSession sharedInstance];
+  // we also need to set default WebRTC audio configuration, since it may be activated after
+  // this method is called
+  RTCAudioSessionConfiguration* config = [RTCAudioSessionConfiguration webRTCConfiguration];
```

Comment:
It's weird to have a different category here as the `webRTCConfiguration` seems to be set to playAndRecord.